{"name": "test_new_customer_a1_error_1", "status": "passed", "description": "新增账户开户失败的用例", "attachments": [{"name": "log", "source": "13659381-2e0c-4eaf-8fd3-78d609ed41ec-attachment.txt", "type": "text/plain"}, {"name": "stderr", "source": "ef22fa07-7a6c-4766-a7ca-26bb239e8987-attachment.txt", "type": "text/plain"}], "start": 1617349321286, "stop": 1617349322896, "uuid": "a9b8f95a-1790-4938-8a26-278ad4ace9cb", "historyId": "712b1c9eb5111c52eb9bcb712e3ccac8", "testCaseId": "e0d1174bec4ade0651120fc5e5340733", "fullName": "tests.test_a6_newclientregister.TestNewCustomerAccount#test_new_customer_a1_error_1", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_a6_newclientregister"}, {"name": "subSuite", "value": "TestNewCustomerAccount"}, {"name": "host", "value": "DESKTOP-U13POE8"}, {"name": "thread", "value": "6480-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_a6_newclientregister"}]}